packages/ethereum-contracts/contracts/apps/SuperTokenV1Library.sol

```diff
@@ -1566,7 +1566,7 @@ library SuperTokenV1Library {
             }
             cfa = IConstantFlowAgreementV1(address(ISuperfluid(host).getAgreementClass(
                 //keccak256("org.superfluid-finance.agreements.ConstantFlowAgreement.v1")
-                    0xa9214cc96615e0085d3bb077758db69497dc2dce3b2b1e97bc93c3d18d83efd3)));
+                0xa9214cc96615e0085d3bb077758db69497dc2dce3b2b1e97bc93c3d18d83efd3)));
```

Comment:
why hardcode the hashes? I would expect this being a trivial optimization compiler will figure out itself.

Reply:
verified that compiler optimization handles this 👍🏼